src/registry.jl

```diff
@@ -99,6 +99,20 @@ add_format(format"OFF", "OFF", ".off", [:MeshIO])
 
 
 
+### Audio formats
+function detectwav(io)
+    seekstart(io)
+    magic = ascii(read(io, UInt8, 4))
+    magic == "RIFF" || return false
```

Comment:
This is probably better as:
```Julia
magic = read(io, UInt8, 4)
magic == b"RIFF" || return false
```
For speed and so that we don't get ascii conversion errors;)